util/topsql/stmtstats/aggregator_test.go

```diff
@@ -84,10 +84,13 @@ func Test_aggregator_run_close(t *testing.T) {
 }
 
 func Test_aggregator_disable_aggregate(t *testing.T) {
```

Comment:
use ```TestAggregatorDisableAggregate```.

Reply:
I'm not sure, this test is for private type, still use big camel case?